internal/queue/queue.go

```diff
@@ -0,0 +1,53 @@
+package queue
+
+import (
+	"os"
+
+	"sublinks/sublinks-federation/internal/log"
+
+	amqp "github.com/rabbitmq/amqp091-go"
+)
+
+type Queue struct {
```

Comment:
Should we call this QueueSrvc/QueueService/??? since it's not really a queue so much as a container for interacting with queues